src/Ztm.WebApi/HttpResponseExceptionFilter.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Newtonsoft.Json;
+
+namespace Ztm.WebApi
+{
+    class ErrorDetails
+    {
+        public object Message { get; set; }
+
+        public override string ToString()
+        {
+            return JsonConvert.SerializeObject(this);
+        }
+    }
+
+    public class HttpResponseExceptionFilter : IActionFilter
+    {
+        public void OnActionExecuting(ActionExecutingContext context)
+        {
+        }
+
+        public void OnActionExecuted(ActionExecutedContext context)
+        {
+            if (context.Exception is HttpResponseException exception)
+            {
+                context.Result = new JsonResult(new ErrorDetails{Message = exception.Value})
```

Comment:
The error response should compliance with [RFC 7807](https://tools.ietf.org/html/rfc7807). See here for example: https://docs.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-2.2#exception-handler